source/guides/accountlink/activating-delegated-harvesting.rst

```diff
@@ -12,162 +12,128 @@ Activating delegated harvesting
+1. Delegate the **main account (M)** importance to a **remote account (R)** using a :ref:`AccountKeyLinkTransaction <account-key-link-transaction>`.
 
-.. mermaid:: ../../resources/diagrams/delegated-harvesting-activation.mmd
-    :caption: Delegated harvesting activation diagram
-    :alt: Delegated harvesting activation diagram
-    :align: center
+2. Link the main account **M** to a **VRF account (V)** for randomized block production and account selection using a :ref:`VrfKeyLinkTransaction <vrf-key-link-transaction>`.
+
+3. Link the main account **M** to a node in order to harvest through that node using a :ref:`NodeKeyLinkTransaction <node-key-link-transaction>`.
+
+4. Request the node to add the remote account **R** as a delegated harvester using a :ref:`PersistentDelegationRequestTransaction <persistent-delegation-request-transaction>`.
+
+Please note that it is entirely up to the node to comply with the request. Moreover, **by design** there is no mechanism to check if the request has been accepted or not, since there is no way to provide such mechanism in a trustless environment.
 
 *************
 Prerequisites
 *************
 
-- Complete :doc:`sending mosaics and messages between two accounts guide <../transfer/sending-a-transfer-transaction>`
+- Complete the :doc:`sending mosaics and messages between two accounts <../transfer/sending-a-transfer-transaction>` guide to have a basic understanding of transactions.
 
-Before you can activate delegated harvesting, you need to have three accounts:
+Before you can activate delegated harvesting, you need the following items:
 
-* **Main account (M)** with at least ``10,000`` harvesting mosaic units.
-* **Announcer account (A)** with enough |networkcurrency|  units to announce a transaction.
-* **Remote account (R)** that did not send or receive any transactions.
+- A **Main account (M)** with at least **10,000** |networkcurrency| to be :ref:`eligible <account_eligibility>` and then some more to pay for transaction fees. This is the account that will receive the harvesting fees. Keep its secret key secret at all times.
 
-*************************
-Method #01: Using the SDK
-*************************
+- A **Remote account (R)** that will act as a proxy between **M** and the node. This account **must have never sent or received any transaction**, and it cannot be involved in any transaction while it is a delegated account.
 
-1. Define your **main account (M)**, **announcer account(A)** and the **remote account (R)** using their private keys.
+- A **VRF account (V)** that has never sent or received any transactions. It is a regular account used to add randomness to the account selection process.
 
-.. example-code::
+- The **node's public TLS key**. This is the key the node uses to authenticate data for transport over TLS and must be provided by the node owner, it cannot be retrieved remotely through any |codename| API.
```

Comment:
For Dual Nodes which is using the catapult-rest >2.2.0, the node's public key can be retrieved from the `node/info` endpoint. For peer only nodes, the key can be extracted from tls connection handshake if the ca can be trusted..

Reply:
I've added this information to the guide.